client/src/components/AutoSuggestInput.js

```diff
@@ -100,19 +88,6 @@ class AutoSuggestInput extends Component {
             <i className="fa fa-search fa-2x" aria-hidden="true" />
           </button>
         </div>
-        <div className="temporary-results-box">
-          { loading && <p> Searching... </p> }
-          { !loading && error && error.searchError && <p>{error.userMessage}</p> }
-          { loaded && nearbyCampaigns && Array.isArray(nearbyCampaigns) &&
-            <div>
-              <p>{"Here's some nearby campaigns:"}</p>
-              {renderNearby(nearbyCampaigns)}
-            </div>
-          }
-          { !loading && nearbyCampaigns && nearbyCampaigns.status === 'okay' && !nearbyCampaigns.results &&
-            <p>{"We didn't find any campaigns near you. Would you like to start one?"}</p>
-          }
-        </div>
       </div>
     );
   }
```

Comment:
Now that we've moved the messaging outside the component, we don't need to pass all those 'InitialSearch' props into this one. Although, error messaging should still happen in this component, so I need to work on that, and I'll take care of that on a different branch.